functions.php

```diff
@@ -39,33 +40,28 @@ function hello_elementor_setup() {
 			add_theme_support( 'title-tag' );
 			add_theme_support(
 				'html5',
-				array(
+				[
 					'search-form',
 					'comment-form',
 					'comment-list',
 					'gallery',
 					'caption',
-				)
+				]
 			);
 			add_theme_support(
 				'custom-logo',
-				array(
+				[
 					'height'      => 100,
 					'width'       => 350,
 					'flex-height' => true,
 					'flex-width'  => true,
-				)
+				]
 			);
 
 			/*
 			 * Editor Style.
 			 */
 			add_editor_style( 'editor-style.css' );
-			
-			/*
-			 * Gutenberg wide images.
-			 */
-			add_theme_support( 'align-wide' );
```

Comment:
Also among the December version changes that need to be included in the PR branch